### PR TITLE
Update rugby's agent structure

### PR DESCRIPTION
**Pull request by rich-nguyen**
This fixes a problem where score updates for a live rugby match were not being reflected on the rugby page.

Added some tests, and changed the agent system so that we can test the rugby job a little easier

### CODE REVIEW

**Comment by janua** [comment]
:+1:
I like the type annotations on everything, I can read it so quickly! :tada:

**Comment by gklopper** [comment]
:+1:

File: sport/app/rugby/jobs/RugbyStatsJob.scala [hunk 1]
```diff
@@ -1,57 +1,60 @@
...
 
-  def liveScores {
-    OptaFeed.getLiveScores.map { matches =>
-      matches.map { aMatch =>
-        val date = dateFormat.print(aMatch.date)
-        val key = s"$date/${aMatch.homeTeam.id}/${aMatch.awayTeam.id}"
-        liveScoreMatches.send {_ :+ aMatch}
-      }
-
+  def sendLiveScores(scoreData: Future[Seq[Match]]) : Future[Any] = {
```
Comment (gklopper) [inline sport/app/rugby/jobs/RugbyStatsJob.scala]: `Future[Any]`?

Comment (rich-nguyen) [inline sport/app/rugby/jobs/RugbyStatsJob.scala]: I wanted it for testing `sendLiveScores`, not that I cared what the return value of the agent send was (which will be set to the value of whatever you passed in)

**Pull request merged**